tests/IndexTest.php

```diff
@@ -798,7 +798,7 @@ public function testForcemerge(): void
 
         $stats = $index->getStats()->getData();
         $this->assertEquals(1, $stats['_all']['primaries']['docs']['count']);
-        $this->assertEquals(0, $stats['_all']['primaries']['docs']['deleted']);
+        $this->assertEquals(2, $stats['_all']['primaries']['docs']['deleted']);
```

Comment:
I'm not entirely sure why before it was 0 and not 1, and now I'm not really sure why it is 2 and not still 1 😕  Anyone knows more here?

Reply:
After a forcemerge deleted document count should probably be `0` I think as it frees up space retaining by deleted document?

Reply:
I'm browsing through the force merge docs and there are quite a few mentions of `deletes`: https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-forcemerge.html But so far could not tie it to this issue we have here.

Reply:
Just make some tests to investigate this problem. I will propose a PR to fix and test with the current version.

Reply:
@ruflin I will push the fix, it's about using `only_expunge_deletes`.

Reply:
Nice, thanks!

Reply:
Done